eclipse/sw360
# fix(test): Use test databases in maven test phase

[//]: # (Copyright Bosch.IO GmbH 2020)
[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.

Added a new `DatabaseSettingsTest` class which will be used for test cases and renamed the `couchdb.properties` to `couchdb-test.properties` in test resources of build-configuration module which is used to keep the test database names and credentials.

> * Which issue is this pull request belonging to and how is it solving it? (*#1125*)
> * Did you add or update any new dependencies that are required for your change? - No

Issue: Closes #1125 

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
> How should these changes be tested by the reviewer?

Need to place `couchdb-test.properties` with test db names and credentials in /etc/sw360 and to verify if it creates and uses the databases with "test" string and uses them for unit test cases.

> Have you implemented any additional tests? - No

### Checklist
Must:
Signed-off-by: Smruti Prakash Sahoo <smruti.sahoo@siemens.com>
